Tasks/Common/AzureRmDeploy-common/azure-arm-rest/AzureServiceClient.ts

```diff
@@ -132,6 +132,10 @@ export class ServiceClient {
                 request.headers["Authorization"] = "Bearer " + token;
                 httpResponse = await webClient.sendRequest(request);
             }
+
+            if(!!httpResponse.headers[CorrelationIdInResponse]) {
+                tl.debug(`Correlation ID from ARM api call response : ${httpResponse.headers[CorrelationIdInResponse]}`);
```

Comment:
The constant CorrelationIdInResponse is not defined in this file. I would suggest you to raise PR in master first before porting changes here so that we dont create more delta in master and releases branch.

Reply:
Done